JSTests/stress/intl-datetimeformat.js

```diff
@@ -254,31 +254,31 @@ shouldBe(Intl.DateTimeFormat('en', { timeZone: 'AMERICA/LOS_ANGELES' }).resolved
 shouldBe(Intl.DateTimeFormat('en', { timeZone: Intl.DateTimeFormat().resolvedOptions().timeZone }).resolvedOptions().timeZone, Intl.DateTimeFormat().resolvedOptions().timeZone);
 
 // Time zone is canonicalized for obsolete links in IANA tz backward file.
```

Comment:
This comment is no longer accurate. We should consider changing it. I’m also not sure we need all these tests if we’re just testing that the same string comes back to us. I wonder what this means about capitalization, like does “australia/darwin” work?

Reply:
Updated the comment. 


>  I wonder what this means about capitalization, like does “australia/darwin” work?

That's covered here

https://github.com/WebKit/WebKit/blob/d9db853f14ab390f3e69c476b831ae89b8e7f120/JSTests/stress/intl-datetimeformat.js#L249-L251